deploy/operator.yaml

```
@@ -41,6 +41,12 @@ spec:
       containers:
         - name: noobaa-operator
           image: NOOBAA_OPERATOR_IMAGE
+          args:
+          - operator         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)